# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/generic/TableCellAccessible.cpp
# Commit: 6d0f4821ddcf
# Full Hash: 6d0f4821ddcfca1b59e4db3befc3ec9c12335891
# Author: James Teh <jteh@mozilla.com>
# Date: 2022-03-23 09:49:32
# Regressor Bug: 1735970
# File Overlap Count: 1
# Description:
#   Bug 1735970 part 3: Add TableAccessibleBase and TableCellAccessibleBase. r=morgan
#   
#   These are needed to support RemoteAccessible tables.
#   Stuff specific to LocalAccessible is still in TableAccessible and TableCellAccessible, which now inherit from the new Base classes.
#   
# ==============================================================================

diff -r d82660fb5408 -r 6d0f4821ddcf accessible/generic/TableCellAccessible.cpp
--- a/accessible/generic/TableCellAccessible.cpp	Wed Mar 23 04:00:25 2022 +0000
+++ b/accessible/generic/TableCellAccessible.cpp	Wed Mar 23 04:00:26 2022 +0000
@@ -12,7 +12,7 @@
 using namespace mozilla;
 using namespace mozilla::a11y;
 
-void TableCellAccessible::RowHeaderCells(nsTArray<LocalAccessible*>* aCells) {
+void TableCellAccessible::RowHeaderCells(nsTArray<Accessible*>* aCells) {
   uint32_t rowIdx = RowIdx(), colIdx = ColIdx();
   TableAccessible* table = Table();
   if (!table) return;
@@ -106,7 +106,7 @@
   return nullptr;
 }
 
-void TableCellAccessible::ColHeaderCells(nsTArray<LocalAccessible*>* aCells) {
+void TableCellAccessible::ColHeaderCells(nsTArray<Accessible*>* aCells) {
   for (LocalAccessible* cell = PrevColHeader(); cell;
        cell = cell->AsTableCell()->PrevColHeader()) {
     aCells->AppendElement(cell);